collect.py

```diff
@@ -306,6 +320,8 @@ def main(bugs):
 
 if __name__ == '__main__':
     random.shuffle(bugs)
+    main(bugs)
```

Comment:
This should be removed. We are already calling the function using multiple threads below.